src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -74,6 +85,18 @@ def map_alternatives_items
       end
     end
 
+    def apply_filters
```

Comment:
apply_filters do not indicate for me that result is iterator with result. Maybe `filtered_alternatives` is better one?